fuel-vm/src/interpreter/executors/instruction.rs

```diff
@@ -41,7 +41,7 @@ where
     }
 
     /// Execute a provided instruction
-    pub fn instruction(&mut self, instruction: Instruction) -> Result<ExecuteState, InterpreterError> {
+    pub fn instruction(&mut self, raw: RawInstruction) -> Result<ExecuteState, InterpreterError> {
```

Comment:
Consider changing this to cleanup a lot of the .into() changes elsewhere:

```suggestion
    pub fn instruction<R: Into<RawInstruction>>(&mut self, raw: R) -> Result<ExecuteState, InterpreterError> {
        let raw = raw.into();
```

Reply:
Done in cb7bed0